data/base-contacts.yaml

```diff
@@ -0,0 +1,3356 @@
+contacts:
```

Comment:
Should this file be stored in the repository? Is the intent for each player to have their own version of this file?